lib/vm/src/testing.rs

```diff
@@ -11,9 +11,11 @@ use cosmwasm::traits::{Api, Storage};
 use cosmwasm::types::{ContractResult, Params, QueryResult};
 
 use crate::calls::{call_handle, call_init, call_query};
+use crate::compatability::check_api_compatibility;
 use crate::instance::Instance;
 
 pub fn mock_instance(wasm: &[u8]) -> Instance<MockStorage, MockApi> {
+    check_api_compatibility(wasm).unwrap();
```

Comment:
That's a very good addition for test data

Reply:
Thanks. Are you fine with making check_api_compatibility VM internal?